### DISCUSSION TITLE
Best Practice converting blurHash to data url

### DISCUSSION THREAD

**Discussion by martin-coded** [bepsvpt/blurhash#8] [General]
What is the best practice to convert the blurhash to a dataURL to use it for example in a Next.js application?

**Comment by bepsvpt** [comment]
If you want to display a BlurHash string on a webpage, you can achieve this using [https://github.com/woltapp/blurhash/tree/master/TypeScript](https://github.com/woltapp/blurhash/tree/master/TypeScript), rather than saving the image as a Data URL. Using a Data URL defeats the main purpose of BlurHash.

You can read more about it in this article: [https://blog.wolt.com/hq/2019/07/01/how-we-came-to-create-a-new-image-placeholder-algorithm-blurhash/](https://blog.wolt.com/hq/2019/07/01/how-we-came-to-create-a-new-image-placeholder-algorithm-blurhash/).
